package.json

```diff
@@ -68,24 +68,26 @@
     "@babel/plugin-transform-runtime": "^7.12.10",
     "@babel/preset-env": "^7.12.11",
     "@babel/preset-typescript": "^7.12.1",
-    "@ceramicnetwork/common": "^0.15.7",
-    "@ceramicnetwork/core": "^0.19.0",
-    "@ceramicstudio/idx-tools": "^0.5.0",
+    "@ceramicnetwork/common": "^0.17.0",
+    "@ceramicnetwork/core": "^0.21.0",
+    "@ceramicstudio/idx-tools": "^0.9.0",
     "@types/elliptic": "^6.4.12",
     "@types/jest": "^26.0.20",
     "@types/node": "^14.14.21",
     "@types/store": "^2.0.2",
     "babel-core": "7.0.0-bridge.0",
     "babel-loader": "^8.2.2",
     "dag-jose": "^0.3.0",
+    "did-resolver": "^3.0.1",
     "eslint": "^7.15.0",
     "eslint-config-3box": "^0.2.0",
-    "ipfs": "^0.52.3",
+    "ipfs": "^0.54.2",
     "it-all": "^1.0.2",
     "jest": "^26.6.2",
     "jest-environment-uint8array": "^1.0.0",
     "jsdoc-babel": "^0.5.0",
     "jsdoc-to-markdown": "^6.0.1",
+    "multiformats": "~4.6.1",
```

Comment:
Is this needed here now?

Reply:
is used in tests, nice to explicitly reference